apps/web/src/components/submission/components/Credential.tsx

```diff
@@ -124,6 +124,8 @@ export const Credential: React.FC = () => {
                       index={index}
                       specialties={specialtyOptions}
                       subspecialties={subspecialties?.[index]}
+                      deleteFunction={() => arrayHelpers.remove(index)}
+                      enableDelete={specialties.length > 1}
```

Comment:
Why do we need this?

Reply:
The delete function handles the removal of the specialty.
The enable delete determines whether of not the delete button is shown. 

I put them at a higher level and pass them down into the component to keep the `SpecialtySelector` component generic and because otherwise I would need to pass the `specialties` array and `arrayHelper` function into the component.

Reply:
The delete function handles the removal of the specialty.
The enable delete determines whether of not the delete button is shown. 

I put them at a higher level and pass them down into the component to keep the `SpecialtySelector` component generic and because otherwise I would need to pass the `specialties` array and `arrayHelper` function into the component.